src/client/index.js

```diff
@@ -18,8 +18,10 @@ export function init ({ appContext, extraContext, pageViewContext }) {
 			...extraContext
 		}),
 		user: getUserData(),
-		// TODO: This should be safe to enable, find out why it hasn't been
-		useSendBeacon: false
+		// Using the Beacon API ensures that no tracking event data is lost 
+		// when the document is being unloaded, which happens when navigating 
```

Comment:
test failure is these trailing spaces :v:

```suggestion
		// Using the Beacon API ensures that no tracking event data is lost
		// when the document is being unloaded, which happens when navigating
```